app/src/main/res/values/strings.xml

```diff
@@ -472,6 +472,7 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="about_category_feedback">Feedback</string>
     <string name="quest_cycleway_value_sidewalk">explicitly shared sidewalk</string>
     <string name="quest_leave_new_note_create_image_error">Unable to create file for photo</string>
+    <string name="no_camera_app">"No external camera app available. Is your pre-installed camera app disabled?"</string>
```

Comment:
This is a little bit long for a toast. I'd suggest to just write "No camera app installed"

Reply:
I took the idea of a longer explanation from https://github.com/streetcomplete/StreetComplete/issues/2911#issuecomment-850585869. Using a shorter sentence is of course also possible.

Reply:
> "No camera app installed"

In #2911 camera app was installed but was not cooperating. If shortening is necessary I would switch to "No camera app available"

Reply:
It would also be possible to replace the `Toast` with a dialog.

Reply:
I'd be for "No camera app available"